packages/core-mobile/app/screens/bridge/utils/bridgeUtils.ts

```diff
@@ -145,20 +60,45 @@ export const networkToBlockchain = (
   }
 }
 
-export const isUnifiedBridgeAsset = (asset: unknown): asset is BridgeAsset => {
-  return asset !== null && typeof asset === 'object' && 'destinations' in asset
-}
-
-export const isUnifiedBridgeTransfer = (
-  transfer?: BridgeTransaction | BridgeTransfer | Transaction
+export const isBridgeTransfer = (
+  transfer?: BridgeTransfer | Transaction
 ): transfer is BridgeTransfer => {
   return transfer !== undefined && 'type' in transfer
 }
 
-export const getNativeTokenSymbol = (chain: Blockchain | Chain): string => {
+export const getNativeTokenSymbol = (chain: Chain): string => {
   if (typeof chain === 'object') {
     return chain.networkToken.symbol
   }
 
   return getNativeSymbol(chain)
 }
+
+export function getOriginalSymbol(symbol: string): string {
+  return symbol.replace(/\.(e|b|p)$/i, '')
```

Comment:
can you add a comment showing an example?

Reply:
```
// get the original symbol without the postfix for network(i.e.: USDC.e -> USDC, BTC.b -> BTC)
```
comment added.